unstructured/ingest/cli/interfaces.py

```diff
@@ -368,12 +368,22 @@ class CliEmbeddingConfig(EmbeddingConfig, CliMixin):
     @staticmethod
     def get_cli_options() -> t.List[click.Option]:
         options = [
+            click.Option(
+                ["--embedding-provider"],
+                help="Type of the embedding class to be used. Can be: "
+                "langchain-huggingface, langchain-openai, or langchain-aws-bedrock.",
```

Comment:
Unless we're setting this as an `ENUM`, lets avoid giving all the options available so that if more get added, the string doesn't become out of date. Or generate that list from the keys in the embedding map we're using and set this as an `ENUM` based on that.

Reply:
I'm dynamically loading the list from the class map keys now. Let me know if you insist on enum inheritance:
[load help message contents dynamically](https://github.com/Unstructured-IO/unstructured/pull/1918/commits/dcb8400e3cd72cc18c106753d60d174f8a589abd)